eth/db/chain.py

```diff
@@ -129,6 +130,34 @@ def persist_block(self,
         with self.db.atomic_batch() as db:
             return self._persist_block(db, block, genesis_parent_hash)
 
+    def persist_unexecuted_block(self,
+                                 block: BlockAPI,
+                                 receipts: Tuple[ReceiptAPI, ...],
+                                 genesis_parent_hash: Hash32 = GENESIS_PARENT_HASH
+                                 ) -> Tuple[Tuple[Hash32, ...], Tuple[Hash32, ...]]:
```

Comment:
Might be nice to have some type aliases for this return type to give them a bit more meaning.  Looking at this it isn't clear to me what these two tuples contain.

Reply:
Sure, no problem. It just took this from the current `persist_block` method.